### ISSUE TITLE
Measure page's frame rate with requestAnimationFrame

### ISSUE THREAD

**Issue by macbre** [macbre/phantomas#405]
``` js
var lastTime = 0; 
function loop(time) { 
 var delay = time – lastTime; 
 var fps = 1000/delay; 
 console.log(delay + ‘ ms’ fps); 
 updateAnimation(); 
 requestAnimationFrame(loop); 
 lastTime = time; 
} 
requestAnimationFrame(loop); 
```

Run it when the page is fully loaded and generate `pageFrameRate` metric from a set of FPS values.

@see https://media.blackhat.com/us-13/US-13-Stone-Pixel-Perfect-Timing-Attacks-with-HTML5-WP.pdf

**Comment by macbre** [comment]
Moving to Roadmap as it needs more research and a thought on how to use it.

**Comment by macbre** [comment]
This can be implemented in Chrome - #707
